Cargo.toml

```diff
@@ -29,9 +29,9 @@ exclude = ["twitch_types", "twitch_oauth2"]
 [workspace.dependencies]
 twitch_api = { version = "0.7.2", path = "." }
 twitch_oauth2 = { version = "0.16.0", path = "twitch_oauth2/" }
-twitch_types = { version = "0.4.9", features = [
+twitch_types = { version = "0.4.10", features = [
     "serde",
-], path = "./twitch_types" }
+] }
```

Comment:
this is wrong

Reply:
Oh yeah I dropped the local path by accident